engine/src/main/java/org/terasology/config/AudioConfig.java

```diff
@@ -31,39 +31,63 @@
 
     private transient PropertyChangeSupport propertyChangeSupport = new PropertyChangeSupport(this);
 
+    /**
+     * @return Returns a float of the sound volume
+     */
     public float getSoundVolume() {
         return soundVolume;
     }
 
+    /**
+     * @param soundVolume Sets the sound volume starting from the previous value and going to the soundVolume passed in
+     */
     public void setSoundVolume(float soundVolume) {
         float oldValue = this.soundVolume;
         this.soundVolume = soundVolume;
         propertyChangeSupport.firePropertyChange(SOUND_VOLUME, oldValue, soundVolume);
     }
 
+    /**
+     * @return Returns music volume
```

Comment:
@returns the music volume as a float, 0 meaning no music and 100 (or 1?) meaning music at the highest volume.

Also, in the previous javadocs it might be worth to write somewhere about "sound" being really "sound effects", as music is also a type of sound.